scaffolds/java_actor_model/src/main/java/com/example/AkkaQuickstart.java

```diff
@@ -0,0 +1,63 @@
+package com.example;
+
+import akka.actor.ActorSystem;
+import akka.actor.AbstractActor;
+import akka.actor.ActorRef;
+import akka.actor.Props;
+
+import java.io.IOException;
+
+
+class MyActor extends AbstractActor {
+    public Receive createReceive() {
+        return receiveBuilder()
+        .match(String.class, p -> {
+            Thread.sleep(5000);
+            System.out.println("This is the String message. The message is: " + p);
+        })
+        .matchAny(p -> {
+            Thread.sleep(5000);
+            System.out.println("The class name of this message is: " + p.getClass().getName());
+        })
+        .build();
+    }
+}
+
+
+public class AkkaQuickstart {
+    public static void main(String[] args) {
+
+        final ActorSystem system = ActorSystem.create("Hello_Akka");
+
+        // Create an actor
+        ActorRef ref1 = system.actorOf(Props.create(MyActor.class), "String_actor");
```

Comment:
Rename `ref<X>` to `actor<X>`, or `<type>Actor`?
```suggestion
        ActorRef stringActor = system.actorOf(Props.create(MyActor.class), "String_actor");
```